src/octopeer-github/test/DatabaseAdaptable/RESTApiDatabaseAdapterTest.ts

```diff
@@ -26,7 +27,7 @@ describe("A RESTApiDatabaseAdapter", function() {
     it("can post to the API", function() {
         const spyFunc = jasmine.createSpy("success");
 
-        adapter.post(new EventObject(defaultElementID, defaultEventID, new Date().getTime(), 100), spyFunc, EMPTY_CALLBACK);
```

Comment:
These changes will cause some merge conflicts with my changes; but it's for a good purpose.